src/app/AppDom.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import MenuDom from '../menu';
+import BodyDom from "../body";
+import {Page} from "../models";
+
+export default class AppDom extends React.Component<Page> {
+    constructor(page:Page){
+        super(page);
+        this.state = {
+            count: 0
+        }
+    }
+
+    render() {
+        return <React.Fragment>
+            <div className="menu">
+                <MenuDom items={this.props.menu.items}/>
```

Comment:
If you don't want to write `this.props....` every time you can declare variables before the `return` statement like this
```
const {menu, body} = this.props;
```
and use them:
```
<MenuDom items={menu.items} />
```

Reply:
updated